backend/battles/tests/test_forms.py

```diff
@@ -15,56 +15,51 @@ def setUp(self):
-                "pokemon_3": self.pokemon_3,
+                "trainer": self.trainer.id,
+                "pokemon_1": mommy.make("pokemon.Pokemon", id=1).id,
+                "pokemon_2": mommy.make("pokemon.Pokemon", id=2).id,
+                "pokemon_3": mommy.make("pokemon.Pokemon", id=3).id,
             },
         }
         form = CreateTeamForm(**params)
-        self.assertTrue(form)
+        self.assertTrue(form.is_valid())
 
-    def test_pokemon_exceeds_points_limit(self):
+    def test_team_cant_have_identical_pokemon(self):
         params = {
-            "initial": {"trainer": self.trainer},
             "data": {
-                "pokemon_1": mommy.make(
-                    "pokemon.Pokemon", id=1, attack=200, defense=200, hp=200
-                ).id,
-                "pokemon_2": mommy.make(
-                    "pokemon.Pokemon", id=2, attack=200, defense=200, hp=200
-                ).id,
-                "pokemon_3": mommy.make(
-                    "pokemon.Pokemon", id=3, attack=200, defense=200, hp=200
-                ).id,
+                "trainer": self.trainer.id,
+                "pokemon_1": mommy.make("pokemon.Pokemon", id=1).id,
+                "pokemon_2": mommy.make("pokemon.Pokemon", id=1).id,
+                "pokemon_3": mommy.make("pokemon.Pokemon", id=2).id,
             },
         }
         form = CreateTeamForm(**params)
         self.assertFalse(form.is_valid())
 
-    def test_send_result_email(self):
+    def test_team_cant_have_invalid_pokemon(self):
         params = {
-            "initial": {"battle": self.battle},
             "data": {
-                "trainer": self.trainer,
-                "pokemon_1": self.pokemon_1.id,
-                "pokemon_2": self.pokemon_2.id,
-                "pokemon_3": self.pokemon_3.id,
+                "trainer": self.trainer.id,
+                "pokemon_1": mommy.make("pokemon.Pokemon", id=-5).id,
+                "pokemon_2": mommy.make("pokemon.Pokemon", id=1).id,
+                "pokemon_3": mommy.make("pokemon.Pokemon", id=2).id,
             },
         }
-        CreateTeamForm(**params)
+        form = CreateTeamForm(**params)
+        self.assertFalse(form.is_valid())
 
-        self.second_trainer = mommy.make("users.User")
+    def test_pokemon_exceeds_points_limit(self):
         params = {
-            "initial": {"battle": self.battle},
             "data": {
-                "trainer": self.second_trainer,
-                "pokemon_1": self.pokemon_1.id,
-                "pokemon_2": self.pokemon_2.id,
-                "pokemon_3": self.pokemon_3.id,
+                "trainer": self.trainer.id,
+                "pokemon_1": mommy.make("pokemon.Pokemon", id=493).id,
+                "pokemon_2": mommy.make("pokemon.Pokemon", id=2).id,
+                "pokemon_3": mommy.make("pokemon.Pokemon", id=3).id,
             },
         }
+        form = CreateTeamForm(**params)
+        self.assertFalse(form.is_valid())
```

Comment:
If you want to also test the validation message raised, you can use [assertRaisesMessage](https://docs.djangoproject.com/en/3.0/topics/testing/tools/#assertions). It would be something like this:

```suggestion
        form = CreateTeamForm(**params)
        self.assertFalse(form.is_valid())
        self.assertFalse(form.is_valid())
        with self.assertRaisesMessage(
                Exception,
                'Trainer, your pokemon team stats can not sum more than ....'):
            form.clean()
```